src/shared/components/lollipopMutationPlot/LollipopPlotNoTooltip.tsx

```diff
@@ -378,13 +463,7 @@ export default class LollipopPlotNoTooltip extends React.Component<LollipopPlotN
                         onClick={this.handlers.onBackgroundClick}
                         onMouseMove={this.handlers.onBackgroundMouseMove}
                     />
-                    <rect
-                        fill="#BABDB6"
-                        x={this.geneX}
-                        y={this.geneY}
-                        height={this.geneHeight}
-                        width={this.props.vizWidth}
-                    />
+                    {this.sequenceSegments}
```

Comment:
We are drawing multiple segments for the gray area under the domains for tooltip purposes...